docs/docs/cmd/teams/user/user-app-remove.mdx

```diff
@@ -14,16 +14,16 @@ m365 teams user app remove [options]
 
 ```md definition-list
 `--id [id]`
-: The unique id of the app instance installed for the user. Specify either `id` or `name`.
+: The unique id of the app instance installed for the user. Specify either `id` or `name` but not both.
```

Comment:
'either/or' is already clearly denoting that users shouldnt use both options. Let's therefore remove the sections 'but not both'